fdbserver/storageserver.actor.cpp

```diff
@@ -9616,14 +9616,77 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 	}
 }
 
+ACTOR Future<Void> createSstFileForCheckPointMetadata(StorageServer* data,
+                                                      CheckpointMetaData metaData,
+                                                      std::string dir) {
+	state int failureCount = 0;
+	loop {
+		try {
+			ASSERT(metaData.ranges.size() > 0);
+			state std::string localFile = dir + "/metadata_bytes.sst";
```

Comment:
Add a const string `checkpointBytesSampleFileName` in `fdbserver/include/fdbserver/RocksDBCheckpointUtils.actor.h`